app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -434,7 +470,16 @@ export default function OnboardingSetup({
         area: ocCityData?.area!,
         region: ocCityData?.region!,
         country: ocCityData?.country!,
-      }).unwrap();
+      })
+        .unwrap()
+        .then(async (res: any) => {
+          await addCityPopulation({
+            cityId: res.data.cityId,
+            locode: res.data.locode!,
```

Comment:
You could just use `data.locode` here, since it didn't change since then, so you don't have to "unwrap" it with the ! operator.